includes/admin_import.php

```diff
@@ -195,6 +195,11 @@ function tsml_import_page()
                         'change_detect' => $data_source_change_detect,
                         'type' => 'JSON',
                     ];
```

Comment:
Moved line to clause where new feed is added, rather than to be run every import sync

Reply:
Excellent. Only shows when a new data source is added.